pipeline_steps/influx.groovy

```diff
@@ -0,0 +1,67 @@
+def setup(){
+  instance_name = "INFLUX"
+  pubCloudSlave.getPubCloudSlave(instance_name: instance_name)
+  common.override_inventory()
+  try{
+    common.conditionalStage(
+      stage_name: "Influx",
+      stage:{
+        withCredentials([
+          file(
+            credentialsId: 'id_rsa_cloud10_jenkins_file',
+            variable: 'jenkins_ssh_privkey'
+          ),
+        ]){
+          dir('/tmp/'){
+            unstash "pubcloud_inventory"
```

Comment:
This will fail with `ERROR: No such saved stash ‘pubcloud_inventory’`

I don't think you need to worry about stashing with this job as all the tasks that don't run on the influx node will run on the same jenkins node. This is due to the top level node() call in the job definition.